test/support/track_sender.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Membrane.RTC.Engine.Support.TrackSender do
```

Comment:
After reading the docs I think I'd somehow include in a name the fact that it's used to send static track from a file, e.g. `StaticTrackSender` or `FixtureTrackSender`